### ISSUE TITLE
korean website adress bug.

### ISSUE THREAD

**Issue by opqriu** [ValvePython/steam#431]
Hi there.
I have sent this issue to steam support.
But the rep told me to report it again on reddit or steam community.
In the end I could not find anyone to answer.
Maybe someone can help me?

![merge_from_ofoct](https://user-images.githubusercontent.com/59023620/225953854-ab84f106-3cfc-4ab0-a62d-72aefd32d2b1.jpg)

Please see the attached screenshot.

1. community guide Click.
2. the pink button at the bottom Click.

When I press this, I don't see guide posts from me.
This is because the web address is "koreana" which is wrong.
So when I change it to "korean", the post looks fine.

I think this is an old bug.
This is an error on the Steam website for all Korean users.
It looks the same on the Guide button in all game communities.
The sentence is weird because you're using deepL.
In other words, the Korean version of the Steam website made the <a href=> weird.
It's incorrectly coded as korean for koreana.
Where and to whom should I tell this?
